frontend/javascripts/admin/admin_rest_api.js

```diff
@@ -876,6 +876,23 @@ export async function startExportTiffJob(
   );
 }
 
+export function startComputeMeshFileJob(
+  organizationName: string,
+  datasetName: string,
+  layerName: string,
+  mag: Vector3,
+  agglomerateView?: string,
+) {
+  // TODO: figure out what's going on with the segmentation layer (only fallback "segmentation" is found by worker)
+  // TODO: figure out why this doesn't work at all... :( Not even with the default example
```

Comment:
@grittaweisheit do you need help with this or could you resolve this together with @fm3 last Friday?

Reply:
We figured out the second one :)
But I'm not sure what's the intended behavior regarding the layer. Should the meshfile always be computed from the fallback (`segmentation`) layer, or is there a case where it should be another layer?

Reply:
I’d say it should always be the segmentation layer (fallback in case of volume annotation, for now), but note that it is not always named `segmentation`. That’s why the named should be passed.